include/boost/geometry/strategies/cartesian/buffer_end_round.hpp

```diff
@@ -116,6 +116,7 @@ public :
                 DistanceStrategy const& distance,
                 RangeOut& range_out) const
     {
+        boost::ignore_unused(perp_left_point);
```

Comment:
Shouldn't there be explicit `#include <boost/core/ignore_unused.hpp>` for this, to keep the header self-contained, etc.?

Reply:
Yes, this is better. Changed. Thanks!

Reply:
Thanks!